x-pack/plugins/lens/public/editor_frame_service/editor_frame/suggestion_panel.scss

```diff
@@ -89,6 +89,12 @@
   flex-grow: 0;
 }
 
-.lnsSuggestionPanel__applyChangesPrompt {
+.lnsSuggestionPanel__applyChangesPrompt.euiPanel {
   height: $lnsSuggestionHeight;
+  background-color: $euiColorLightestShade;
```

Comment:
This is just an `EuiPanel color="subdued"` no need for extra styling to beat out native ones.. 😏

Reply:
I believe this is intentional. The `color="subdued"` prop for `EuiPanel` actually uses `#FAFBFD` for the background color, which is `$euiPageBackgroundColor`. It’s ever-so-slightly lighter than `#F5F7FA`, which is `$euiColorLightestShade`.

Reply:
Not quite! See it [here](https://github.com/elastic/eui/blob/main/src/global_styling/variables/_panel.scss#L15).